pyproject.toml

```diff
@@ -52,9 +52,9 @@ maintainers = [
 # Note: Python and NumPy upper version bounds should be set correctly in
 # release branches, see:
 #     https://scipy.github.io/devdocs/dev/core-dev/index.html#version-ranges-for-numpy-and-other-dependencies
-requires-python = ">=3.8"
+requires-python = ">=3.8,<3.11"
 dependencies = [
-    "numpy>=1.17.3",
+    "numpy>=1.17.3,<1.24.0",
```

Comment:
`1.22.0rc1` is out and we're clean on deprecations there, so we can make this `<1.25.0`.